config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -0,0 +1,91 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:55680
+      http:
+        endpoint: 0.0.0.0:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+  awsecscontainermetrics:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+  batch/metrics:
+    timeout: 60s
+  filter:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - ecs.task.memory.reserved
+          - ecs.task.memory.utilized
+          - ecs.task.cpu.reserved
+          - ecs.task.cpu.utilized
+          - ecs.task.network.rate.rx
+          - ecs.task.network.rate.tx
+          - ecs.task.storage.read_bytes
+          - ecs.task.storage.write_bytes
+  metricstransform:
+    transforms:
+      - metric_name: ecs.task.memory.utilized
+        action: update
+        new_name: MemoryUtilized
+      - metric_name: ecs.task.memory.reserved
+        action: update
+        new_name: MemoryReserved
+      - metric_name: ecs.task.cpu.utilized
+        action: update
+        new_name: CpuUtilized
+      - metric_name: ecs.task.cpu.reserved
+        action: update
+        new_name: CpuReserved
+      - metric_name: ecs.task.network.rate.rx
+        action: update
+        new_name: NetworkRx
+      - metric_name: ecs.task.network.rate.tx
+        action: update
+        new_name: NetworkTx
+      - metric_name: ecs.task.storage.read_bytes
+        action: update
+        new_name: StorageRead
+      - metric_name: ecs.task.storage.write_bytes
+        action: update
+        new_name: StorageWrite
+exporters:
+  awsxray:
+  awsemf/application:
+    log_group_name: aws/ecs/containerinsights/{{ClusterName}}/application/metrics
+    log_stream_name: {{TaskId}}
+  awsemf/performance:
+    namespace: ECS/ContainerInsights
+    log_group_name: /aws/ecs/containerinsights/{{ClusterName}}/performance
+    log_stream_name: {{TaskId}}
+    resource_to_telemetry_conversion:
+      enabled: true
+    dimension_rollup_option: NoDimensionRollup
+    metric_declarations:
+      dimensions: [ [ aws.ecs.cluster.name ], [ aws.ecs.cluster.name, aws.ecs.task.family ] ]
```

Comment:
I think the dimensions need to be renamed- we need them to be called `ClusterName` and `TaskDefinitionFamily` for container insights. @hossain-rayhan

Reply:
Yea, we need an option for renaming our resource attributes which are being converted to metric labels later. That's what I mentioned in the doc- Right now it takes two actions in the `resourceprocessor` to rename an attribute. Either we can go with this existing approach or we can wait for the new proposed feature. I had a discussion with OT maintainer Tigran on gitter and also discussed this in the Metrics SIG meeting. Please check the issue here- [2184](https://github.com/open-telemetry/opentelemetry-collector/issues/2184).

Reply:
I thought it again- going forward with the current approach would be ideal. Once the new feature is available, we can update that without introducing customer facing changes.

I will test and update the templates.

Reply:
This still needs to be updated in this template right?